pkgs/development/python-modules/torch/fix-cmake-cuda-toolkit.patch

```diff
@@ -0,0 +1,1128 @@
+From 2533fb7225d030e719630dfa2e47eda94ef69668 Mon Sep 17 00:00:00 2001
+From: Gaetan Lepage <gaetan@glepage.com>
+Date: Thu, 23 Nov 2023 00:39:26 +0100
+Subject: [PATCH] fix-cmake-cuda-toolkit
+
+---
+ CMakeLists.txt                      |    4 -
+ cmake/Modules/FindCUDAToolkit.cmake | 1073 ---------------------------
+ cmake/public/cuda.cmake             |    4 +-
+ 3 files changed, 2 insertions(+), 1079 deletions(-)
+ delete mode 100644 cmake/Modules/FindCUDAToolkit.cmake
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3a48eaf4e29..3aaeef2593a 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1152,10 +1152,6 @@ if(BUILD_SHARED_LIBS)
+       ${PROJECT_SOURCE_DIR}/cmake/Modules_CUDA_fix
+       DESTINATION share/cmake/Caffe2/
+       COMPONENT dev)
+-  install(FILES
+-      ${PROJECT_SOURCE_DIR}/cmake/Modules/FindCUDAToolkit.cmake
+-      DESTINATION share/cmake/Caffe2/
+-      COMPONENT dev)
+   install(FILES
+       ${PROJECT_SOURCE_DIR}/cmake/Modules/FindCUSPARSELT.cmake
+       DESTINATION share/cmake/Caffe2/
+diff --git a/cmake/Modules/FindCUDAToolkit.cmake b/cmake/Modules/FindCUDAToolkit.cmake
+deleted file mode 100644
+index 7c8a79c5493..00000000000
+--- a/cmake/Modules/FindCUDAToolkit.cmake
++++ /dev/null
+@@ -1,1073 +0,0 @@
+-
+-# This module is back-ported from CMake 3.17 and above to work with CMake 3.10
```

Comment:
Ouch. On the second thought, we probably want to keep `rm cmake/Modules/FindCUDAToolkit.cmake` as a postPatch command. That way we won't get any conflicts if upstream modifies their vendored module (we don't want to, because we always wan to use the kitware's module), but we would get one if they change how it's used (we want to get those, because they signal that something relevant might have changed). We'd also keep the diff smaller